src/debugger/symbols.c

```diff
@@ -16,11 +17,13 @@ struct mDebuggerSymbol {
 
 struct mDebuggerSymbols {
 	struct Table names;
+	struct Table reverse;
```

Comment:
This PR is missing the cleanup from this reverse table (including removing symbols).